test/e2e/advanced_test.go

```diff
@@ -0,0 +1,87 @@
+package e2e_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions/deploy"
+	kubecli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/cli/kubecli"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/model"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+)
+
+var _ = Describe("Deploy cluster", Label("cluster-ns-ct"), func() {
+	var data model.TestDataProvider
+
+	BeforeEach(func() {
+		Eventually(kubecli.GetVersionOutput()).Should(Say(K8sVersion))
+	})
+	AfterEach(func() {
+		GinkgoWriter.Write([]byte("\n"))
+		GinkgoWriter.Write([]byte("===============================================\n"))
+		GinkgoWriter.Write([]byte("Operator namespace: " + data.Resources.Namespace + "\n"))
+		GinkgoWriter.Write([]byte("===============================================\n"))
+		if CurrentSpecReport().Failed() {
+			GinkgoWriter.Write([]byte("Test has been failed. Trying to save logs...\n"))
+			utils.SaveToFile(
+				fmt.Sprintf("output/%s/operatorDecribe.txt", data.Resources.Namespace),
+				[]byte(kubecli.DescribeOperatorPod(data.Resources.Namespace)),
+			)
+			utils.SaveToFile(
+				fmt.Sprintf("output/%s/operator-logs.txt", data.Resources.Namespace),
+				kubecli.GetManagerLogs(data.Resources.Namespace),
+			)
+			actions.SaveTestAppLogs(data.Resources)
+			actions.SaveK8sResources(
+				[]string{"deploy", "atlasclusters", "atlasprojects"},
+				data.Resources.Namespace,
+			)
+			actions.AfterEachFinalCleanup([]model.TestDataProvider{data})
+		}
+	})
+
+	DescribeTable("Namespaced operators working only with its own namespace with different configuration",
+		func(test model.TestDataProvider) {
+			data = test
+			advancedMainCycle(test)
+		},
+		Entry("Trial - Simplest configuration with no backup and no user", Label("ns-trial-ct"),
```

Comment:
`ct` in the label, we need to rename probably

Reply:
Thanks for your review
Changed it to `ns-crd` what do you think!?
https://github.com/commercetools/mongodb-atlas-kubernetes/commit/fb092d29b2e9876fc94865a60a18a14c70cceaf5#diff-61b8c9f07e0e7ff9a7b7de47dcb124b58281557b37b9ff3b88db086492c90da9R53